netci.groovy

```diff
@@ -585,7 +585,7 @@ branchList.each { branchName ->
             if (isPR) {
                 // Set PR trigger.
                 // Set of OS's that work currently. 
-                if (os in ['OSX', 'Ubuntu', 'OpenSUSE13.2', 'CentOS7.1']) {
+                if (os in ['OSX', 'Ubuntu15.10', 'OpenSUSE13.2', 'CentOS7.1']) {
```

Comment:
I don't think we want to do Ubuntu 15.10 per PR, the coverage from 14.04 + Centos should be sufficient?